ArduPlane/Parameters.cpp

```diff
@@ -1211,11 +1211,16 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("DSPOILER_CROW_W2", 18, ParametersG2, crow_flap_weight2, 0),
     
+    // @Group: NAVLQR_
+    // @Path: ../libraries/AP_LQR_Control/AP_LQR_Control.cpp
+    AP_SUBGROUPINFO(lqr_control, "NAVLQR_", 19, ParametersG2, AP_LQR_Control),
+    
     AP_GROUPEND
 };
 
 ParametersG2::ParametersG2(void) :
-    ice_control(plane.rpm_sensor, plane.ahrs)
+    ice_control(plane.rpm_sensor, plane.ahrs),
+    LQR_controller(plane.ahrs, plane.TECS_controller)
```

Comment:
First parameter is not needed at all as AHRS is available as singleton.
As for the error you are getting, it is because you have AP_LQR_Control(AP_AHRS &ahrs, const AP_SpdHgtControl *spdHgtControl) in "h" file (reference and pointer) but you try to init it with two references in Parameters.cpp. If you keep pointer for spdHgtControl, it should be inited as &plane.TECS_controller in Parameters.cpp.

Reply:
Thanks. It solved the issue. This branch is compiling successfully on my system. I hope it would build on yours too.